ichikoto/settings.py

```diff
@@ -158,6 +158,15 @@
 else:
     EMAIL_SCHEDULER_MINUTES = 0
 
+# Grabs the users target language
+language = os.getenv("TARGET_LANGUAGE")
+available_languages = ["Japanese", "Spanish"]
+
+if language.capitalize().strip() in available_languages:
+    TARGET_LANGUAGE = language.capitalize().strip()
+else:
+    TARGET_LANGUAGE = available_languages[0]
+
```

Comment:
well planned, meant to add an issue to standardize the language names, nice!

Reply:
Yeah I figured you'd only want to allow languages that have already been through some sort of PR/QA stage, so..